### DISCUSSION TITLE
AskPDF读不了pdf

### DISCUSSION THREAD

**Discussion by piliwu** [MuiseDestiny/zotero-gpt#115] [Q&A]
更新到最新版后AskPDF不能用了，重新设置了秘钥也还是不行。其他功能可以用。
<img width="680" alt="屏幕截图 2023-04-22 102750" src="https://user-images.githubusercontent.com/89573308/233754704-c123031b-3a28-447f-acb8-776b0b629219.png">
贴一个秘钥设置截图，我是国外ip，不用梯子。之前的版本是好使的，就是更新了最新版之后有的问题。
<img width="404" alt="屏幕截图 2023-04-22 104027" src="https://user-images.githubusercontent.com/89573308/233755276-c8767ec9-d489-49bb-8b40-31c80b57e5cf.png">

**Comment by AbigaleLu** [comment]
> 更新到最新版后AskPDF不能用了，重新设置了秘钥也还是不行。其他功能可以用。
> 
> <img width="680" alt="屏幕截图 2023-04-22 102750" src="https://user-images.githubusercontent.com/89573308/233754704-c123031b-3a28-447f-acb8-776b0b629219.png">
> 
> 贴一个秘钥设置截图，我是国外ip，不用梯子。之前的版本是好使的，就是更新了最新版之后有的问题。
> 
> <img width="404" alt="屏幕截图 2023-04-22 104027" src="https://user-images.githubusercontent.com/89573308/233755276-c8767ec9-d489-49bb-8b40-31c80b57e5cf.png">
> 
> 
> 
> 

之前是不是更改过里面的内容呀🧐

**Reply by MuiseDestiny** [reply]
右键长按askpdf

**Reply by jhs1965net** [reply]
Hi MuiseDestiny , really appreciate your work , it save me a big amount of time and one my research.
Sorry but I can't open get with the hot keys you give , / is in number 7 and ? is in' , Spanish keyboard.
Thanks a lot for your great work

James

**Reply by hkhcy** [reply]
#🪐AskPDF[position=10][color=#0EA293][trigger=/^(本文|这篇文章|论文)/]
You are a helpful assistant. Context information is below.
<code>
Meet.Global.views.messages = [];
Meet.Zotero.getRelatedText(Meet.Global.input)
</code>
Using the provided context information, write a comprehensive reply to the given query. Make sure to cite results using [number] notation after the reference. If the provided context information refer to multiple subjects with the same name, write separate answers for each subject. Use prior knowledge only if the given context didn't provide enough information.

Answer the question: <code>Meet.Global.input</code>

Reply in zh-CN

这样是否是正确的呢？

**Comment by Jack-Liu0227** [comment]
我也出现了这个问题，出现了以下错误，有解决了的吗？蹲一下
context_length_exceeded

https://api.chatanywhere.com.cn/v1/chat/completions

invalid_request_error
OpenAI response error

**Comment by Lee201612** [comment]
在插件设置中，general下面有embeding，把里面的openai的api换成chatanywhere的
